### DISCUSSION TITLE
Is the browser window size a parameter for generating the fingerprint?

### DISCUSSION THREAD

**Discussion by Chevindu** [fingerprintjs/fingerprintjs#878] [Q&A] [answered]
Hi, 
I saw in one of the discussions that the user's screen resolution and browser's frame coordinates (positioning I assume) are considered as parameters for generating the fingerprint. I was wondering if the browser's window size is also a parameter for this. I hope it's not a foolish question, because I am not aware of the technology behind this. 
Thank you.

**Comment by JuroUhlar** [comment, marked as answer]
HI @Chevindu! It's not a foolish question at all :) 

Screen resolution is one of the signals used for fingerprinting, but window size and position are not. You can have a look at the FingerprintJS demo: https://fingerprintjs.github.io/fingerprintjs/ 
which shows a JSON of all the data that is hashed into a fingerprint. 

You can try resizing your browser window and reloading the demo page, you will see that the fingerprint does not change. However, dragging the browser window to a second monitor with a different screen resolution than the first one will change the fingerprint. 

The `visitorId` provided by Fingerprint Pro (as opposed to the open-source Fingerprint JS) is much more stable: https://fingerprint.com/demo/
